docs/docusaurus.config.js

```diff
@@ -358,7 +358,7 @@ const config = {
               {
                 html: `<div class="footer-links">
                   <span>© ${new Date().getFullYear()} Langflow</span>
-                  <span id="preferenceCenterContainer"> ·&nbsp; <a href="/cookies">Manage Privacy Choices</a></span>
+                  <span id="preferenceCenterContainer"> ·&nbsp; <a href="https://langflow.org/preferences">Manage Privacy Choices</a></span>
```

Comment:
`#nitpick` - should we just use `/preferences` here?
```suggestion
                  <span id="preferenceCenterContainer"> ·&nbsp; <a href="/preferences">Manage Privacy Choices</a></span>
```

Reply:
No, because that would go to docs.langflow.org/preferences